api/MediaSession.json

```diff
@@ -299,6 +309,68 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "MediaPositionState": {
+          "__compat": {
+            "description": "Dictionary passed to <code>setPositionState</code> to specify play position within a track (<code>{duration, playbackRate, position}</code>).",
```

Comment:
Yeah, I see what you mean here. IIRC, you had subfeatures for duration, playbackRate, position in your original PR (that data seems to have disappeared with a rebase, so I don't know this for a fact). Seeing this singular `MediaPositionState` feature though, without anything underneath, I don't think it adds anything to the parent feature—it's effectively saying, "this method has an argument, which is an object".

My inclination here is to drop this feature and move on.

Reply:
Completely agree. Done!